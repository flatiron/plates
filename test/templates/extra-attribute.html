<img src="" class="url extra"></img>
