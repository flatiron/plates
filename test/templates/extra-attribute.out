<img src="http://example.com" class="url extra"></img>
